.env.development

```diff
@@ -1,5 +1,6 @@
 BASE_URL='localhost:8734'
-ENTERPRISE_CATALOG_MFE_URL=''
+ENTERPRISE_CATALOG_MFE_URL='http://example.com'
+ORDERS_MFE_URL='http://localhost:1996'
```

Comment:
`ORDERS_MFE_URL` for development is the port that the frontend-app-ecommerce MFE runs on. Note: If this is the right approach for how we manage what that URL is, we'll need to add this environment variable to edx-internal so it's part of stage/prod builds.